backend/src/main/java/com/bakdata/conquery/models/preproc/parser/specific/CompoundDateRangeParser.java

```diff
@@ -0,0 +1,59 @@
+package com.bakdata.conquery.models.preproc.parser.specific;
+
+import com.bakdata.conquery.models.common.daterange.CDateRange;
+import com.bakdata.conquery.models.config.ConqueryConfig;
+import com.bakdata.conquery.models.events.stores.root.DateRangeStore;
+import com.bakdata.conquery.models.events.stores.specific.DateRangeTypeCompound;
+import com.bakdata.conquery.models.exceptions.ParsingException;
+import com.bakdata.conquery.models.preproc.parser.ColumnValues;
+import com.bakdata.conquery.models.preproc.parser.Parser;
+import lombok.ToString;
+import lombok.extern.slf4j.Slf4j;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * CompoundDateRangeParser is the Parser of {@link CDateRange} whose start and end should be saved as references
+ * It does not have an implementation logic. It only creates his parser and ignores his input values
+ */
+@Slf4j
+@ToString(callSuper = true)
+public class CompoundDateRangeParser extends Parser<CDateRange, DateRangeStore> {
+
+	@NotNull
+	final private String startColumn, endColumn;
```

Comment:
Kannst du dir bitte ein bisschen Zeit nehmen und schauen, ob du in der OutputDescription eine Validation machen kannst, dass start und endColumn existieren? Wenn du nichts offensichtliches findest ist das kein Problem. Würde aber vermutlich auch wieder über JsonBackreference funktionieren. Dadurch können wir beim Preprocess sicher stellen, dass das kein Müll ist.

Reply:
Guter Ansatz aber bitte ValidationMethod annotation benutzen.